### ISSUE TITLE
Shouldn't report assertion to any

### ISSUE THREAD

**Issue by kke** [gostaticanalysis/fourcetypeassert#17]
It's not often, if ever, you need to do:

```go
  objAny := obj.(any)
```

But when you do, forcetypeassert wants you to check it.
